bindings/stronghold-nodejs/Cargo.toml

```
@@ -1,5 +1,5 @@
 [package]
-name = "stronghold-nodejs"
+name = "identity-stronghold-nodejs"
 version = "0.0.0"
 edition = "2021"
 
```

Aside: prevent publishing Rust source from bindings.

```suggestion
name = "identity-stronghold-nodejs"
version = "0.0.0"
edition = "2021"
publish = false
```